### ISSUE TITLE
Space dashboard visual "pop"/re-layout on each visit

### ISSUE THREAD

**Issue by andrewazores** [openshiftio/openshift.io#4052]
When visiting the space dashboard view, there is a noticeable "pop" or "flicker" of the page content which appears to be multiple items' layouts changing and the view content redrawing. This looks pretty jarring. I suspect this is a feature-flags thing.

![ezgif-5-92f21e1b67](https://user-images.githubusercontent.com/3787464/43100253-c0bada4e-8e92-11e8-9c7e-b9e566f6bef6.gif)

**Comment by aptmac** [comment]
Yeah, in the gif you can see both the production/beta level and exp/internal mastheads being rendered one after the other.
